docs/topics/wildfly-swarm/docs/reference/fractions/keycloak.adoc

```diff
@@ -27,6 +27,9 @@ Base URL of the Realm Auth Server
 swarm.keycloak.realms._KEY_.auth-server-url-for-backend-requests:: 
 URL to use to make background calls to auth server
 
+swarm.keycloak.realms._KEY_.autodetect-bearer-only:: 
```

Comment:
This whole file needs to be consistent with its list elements. Either make them all full sentences with capitals and periods or make them all phrases with lowercase and no periods.

Reply:
Note to self: not easily fixable in our sources.